rustls-platform-verifier/src/tests/verification_mock/mod.rs

```diff
@@ -116,47 +111,47 @@ fn test_verification_without_mock_root() {
 // Verifies that our test trust anchor(s) are not trusted when `Verifier::new()`
 // is used.
 mock_root_test_cases! {
-    valid_no_stapling_dns [ any(windows, target_os = "android", target_os = "macos", target_os = "linux") ] => TestCase {
+    valid_no_stapling_dns [ any(windows, unix) ] => TestCase {
```

Comment:
It's not clear from the diff but we still require the `cfg` portion of the `mock_root_test_cases!` macro for the `stapled_revoked_xxx` cases that are still gated to only Windows, Android or MacOS, they aren't _all_ `any(windows, unix)`.